### ISSUE TITLE
Custom DNS

### ISSUE THREAD

**Issue by Empty2k12** [LuckyTurtleDev/crab-hole#28]
Pihole has the option of adding custom DNS entries. I would like to add internal DNS names to my private crab-hole.

![image](https://github.com/LuckyTurtleDev/crab-hole/assets/6954968/e7a6cb77-d397-4e2e-8769-a315db4722e6)
Image per https://discourse.pi-hole.net/t/howto-using-pi-hole-as-lan-dns-server/533

**Comment by msrd0** [comment]
Should be fairly easy to implement, one would just need to insert the local entries into the catalog here: https://github.com/LuckyTurtleDev/crab-hole/blob/main/src/main.rs#L144

**Comment by Empty2k12** [comment]
Precisely. It would probably be best served as a separate config file with DNS name <=> IP mappings.
